spec/system/index_spec.rb

```diff
@@ -67,18 +65,18 @@
     end
     it 'displays a helpful message' do
       expect(page).to have_selector 'p.notice',
-                                    text: 'Stop 1234 not found'
+                                    text: "Stop #{incorrect_stop_name} not found"
     end
   end
   context 'without completing stop name' do
-    let!(:bus_stop) { create :bus_stop, name: 'Mill Valley Apartments' }
     it 'autofills' do
-      fill_in 'Enter stop name', with: 'Mill Valley'
-      find('div', text: 'Mill Valley Apartments').click
+      # use the first character of the stop name so it's incomplete
```

Comment:
What do people think of this comment?

Reply:
... so that it can be completed by the autocomplete, right?

Reply:
Dammit thanks dave